.cirrus.yml

```diff
@@ -50,6 +50,9 @@ global_task_template: &GLOBAL_TASK_TEMPLATE
   depends_built_cache:
     folder: "depends/built"
     fingerprint_script: echo $CIRRUS_TASK_NAME $(git rev-list -1 HEAD ./depends)
+
+global_task_template: &GLOBAL_TASK_TEMPLATE
+  << : *CONTAINER_DEPENDS_TEMPLATE
   << : *MAIN_TEMPLATE
 
 macos_native_task_template: &MACOS_NATIVE_TASK_TEMPLATE
```

Comment:
unrelated: What is the point of a template that is only used once? We don't have a template for windows either.

Reply:
There is no point now. The initial idea was to re-use it with two Apple tasks: x86_64 and arm64.

Reply:
nit: Mind inlining it here?

Reply:
(in a new commit)

Reply:
> nit: Mind inlining it here?

> (in a new commit)

Done.